### PR TITLE
[ADD] user_log_view

### PR DESCRIPTION
**Pull request by trojikman**
Solves this issue https://github.com/OCA/server-auth/issues/154

### CODE REVIEW

File: user_log_view/views/res_users_views.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+<odoo>
+
+<record id="res_users_view_form" model="ir.ui.view">
+    <field name="name">res.users.view.form</field>
+    <field name="model">res.users</field>
+    <field name="inherit_id" ref="base.view_users_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//header" position="inside">
+            <button name="action_user_log" type="object" string="Authentication logs" class="oe_read_only"/>
```
Comment (yelizariev) [inline user_log_view/views/res_users_views.xml]: Make smart button instead

**Comment by yelizariev** [comment]
Consider to add grouping by date in tree view

File: user_log_view/views/res_users_views.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <field name="target">current</field>
+</record>
+
+<record id="res_users_view_form" model="ir.ui.view">
+    <field name="name">res.users.view.form</field>
+    <field name="model">res.users</field>
+    <field name="inherit_id" ref="base.view_users_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//button[hasclass('oe_stat_button')]" position="after">
+            <button name="user_log_view.action_user_log" type="action" string="Authentication logs" icon="fa-th-list" />
```
Comment (yelizariev) [inline user_log_view/views/res_users_views.xml]: What about something less boring? E.g. https://fontawesome.com/icons/user-secret?style=solid

**Review by yelizariev — APPROVED** [review]

**Comment by yelizariev** [comment]
Screenshots

screenshot-3404838-155-276821 runbot2 odoo-community org-2020 01 15-15_07_53
---
screenshot-3404838-155-276821 runbot2 odoo-community org-2020 01 15-15_08_52

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**